### PR TITLE
feat: ability to filter ILC Routes in registry by route string

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: registry/client/src/appRoutes/List.js [hunk 1]
```diff
@@ -72,6 +73,11 @@ const ListFilter = ({ routerDomain, ...props }) => {
                     />
                 : null
             }
+            {
+                !props.filterValues.showSpecial
+                ? <TextInput label="Route prefix" source="routePrefix" alwaysOn resettable/>
```
Comment (nightnei) [inline registry/client/src/appRoutes/List.js:78]: why do you use word "prefix"? I think label can be just "Route" and source="route"

Comment (nightnei) [inline registry/client/src/appRoutes/List.js:78]:
```suggestion
                ? <TextInput label="Route" source="route" alwaysOn resettable/>
```

File: registry/server/appRoutes/routes/getAppRoutes.ts [hunk 2]
```diff
@@ -16,6 +16,10 @@ const getAppRoutes = async (req: Request, res: Response) => {
         query.where('routes.route', 'like', `${SPECIAL_PREFIX}%`);
     } else {
         query.whereNot('routes.route', 'like', `${SPECIAL_PREFIX}%`);
+
+        if (filters.routePrefix !== undefined) {
+            query.where('routes.route', 'like', `${filters.routePrefix}%`)
```
Comment (nightnei) [inline registry/server/appRoutes/routes/getAppRoutes.ts:21]: and here it's better to allow whole search with "%" at the beginning

```suggestion
            query.where('routes.route', 'like', `%${filters.route}%`)
```

**Review by nightnei — CHANGES_REQUESTED** [review]

File: registry/client/src/appRoutes/List.js (hunk 1 above)
Comment (nightnei) [reply registry/client/src/appRoutes/List.js:78]: why do you use word "prefix"? I think label can be just "Route" and source="route"

**Review by nightnei — APPROVED** [review]

File: registry/client/src/appRoutes/List.js (hunk 1 above)
Comment (StyleT) [reply registry/client/src/appRoutes/List.js:78]: To emphasise that it is a prefix based search. Ex:
/foo maches: /foo, /foo/ & /foo/bar but doesn't match /test/foo

**Pull request merged**